projects/igniteui-angular/src/lib/core/styles/themes/_core.scss

```diff
@@ -33,6 +33,7 @@
 @import '../components/carousel/carousel-component';
 @import '../components/checkbox/checkbox-component';
 @import '../components/chip/chip-component';
+@import '../components/column-actions/column-actions-component';
 @import '../components/column-hiding/column-hiding-component';
```

Comment:
If you are to copy the column hiding theme, then we should also remove the column hiding source code as it's essentially just more of the same.